spring-cloud-gcp-dependencies/pom.xml

```diff
@@ -19,6 +19,7 @@
 		<cloud-trace-java.version>0.5.0</cloud-trace-java.version>
 		<spring-cloud-context.version>1.2.3.RELEASE</spring-cloud-context.version>
 		<spring-cloud-sleuth.version>1.2.5.RELEASE</spring-cloud-sleuth.version>
+        <spring-cloud-stream.version>1.3.0.RELEASE</spring-cloud-stream.version>
```

Comment:
May someone here explain me why can't we upgrade to Spring Cloud/Spring Boot 2.0 yet?
Right, they are still under development, but we are too. And we really don't have plans for real GA here in the nearest future. That is absolutely fine to depend on those brand new versions and pull the best from all the Spring portfolio.
More over that would be great feedback from our side to their since we are all still develop something.

The situation with this Binder that Spring Cloud Stream 2.0 is slightly different than 1.3 so we again will have some work in the future when we upgrade anyway. Well, that's fine too if we release SC-GCP-2.0, but when it is going to happen? I think people will just lose an interest to our project if it isn't aligned with the current Spring IO state. No body will be interested to use Spring Boot 1.5 based project when Spring Boot 2.0 is here.

Just my five fingers why I don't review this PR.

Thanks for understanding.

/CC @markfisher , @garyrussell , @olegz

Reply:
Of course we want to be on par with what's out there, but I personally prefer against doing the change in this PR, since it's dragged on for way too long now (e.g., months) and it will drag on even more.

That sounds very much like the kind of change that we can easily do outside of this PR (#201). In the meantime, we would also be seeing feedback from prospective users that we otherwise wouldn't if the PR isn't merged.

Reply:
My point of view if we review/merge this now, we will be forced to review/merge one more version when we upgrade to SCSt 2.0.

 The feature is still fresh and why don't do it only once?

Reply:
I think we should carefully consider the decision to move (or branch) to Spring Boot 2.0 because it could affect many other parts of the project. We definitely want to tap into as many users as possible, and I think most of the users are on 1.5 right now. That being said, we'll need to move to 2.0 at some point, but I would hope that it wouldn't hold back this PR. I hope whatever changes are in SCSt 2.0, most of the code would still be the same and the effort spent reviewing this wouldn't be all wasted.

Let's discuss more on the next call.

Reply:
Well, IMO those users who can use our milestone may easily use Spring Boot 2.0 and so on.
More over most Spring project are already at GA for Spring Framework 5.0 base line and developers from there really encourage people to use the latest version for new projects.
I don't know why you try to stick with an old version for this new project when our real GA is far away the next year. That is absolutely fine to stay in line with modern trends.

OK. I don't have more arguments to convince you do not do the same work twice, but if you still insist in SF-4.3 base line for our version 1.0, let it.

I would be glad to hear other points of you of my team members, but as long as I am in development for AWS Kinesis Binder I can say that there are a lot of critical changes in SCSt 2.0 so we decided to move it directly to Spring Boot 2.0 to avoid extra work with the upcoming features.

Reply:
I don't mind doing the switchover to SB2.0 right after this PR.

I would just rather do this work with other PRs than on this one. This one is as good as done anyway, as far as initial implementation goes.

Reply:
Correct. I don't request to do Boot 2.0 work here in this PR.
I suggest to do that in the separate isssue/PR.

Merge it and be scared how much work we have to do here for SCSt 2.0 compatibility yet. 😄 

That's just my feeling though: there are a lot of interesting tasks to do rather than always redo not released code yet, just because we have started not from proper version...

I think I have already exhausted you with this subject, so, I will start to do what is easier for you and what you will say is the best to go. I might just don see the whole picture with the market around and insist in the moving on according my experience in other projects.